Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -823,7 +826,7 @@ def audit_specs
       stable_url_minor_version = stable_url_version.minor.to_i
 
       formula_suffix = stable.version.patch.to_i
-      throttled_rate = THROTTLED_FORMULAE[formula.name]
+      throttled_rate = tap_audit_exception_list(:throttled_formulae)[formula.name]
```

Comment:
```suggestion
      throttled_rate = tap_audit_exceptions(:throttled_formulae, formula.name)
```

feels like a nicer API to me.